### ISSUE TITLE
Blinking bar

### ISSUE THREAD

**Issue by juster-0** [florentc/xob#42]
The bar sometimes "blinking" while changing values.

https://user-images.githubusercontent.com/18105077/179231174-16451acc-d076-4c47-bbaf-de40b253ac38.mov

After checking the video frame by frame, I've noticed that blinking is the result of redrawing the bar every time, when values change.

To solve the issue we can redraw only changed part of the bar. And draw all components of the bar separately.

**Comment by florentc** [comment]
This may be related to #37. Could you confirm the bug is introduced by commit ff4911e066f4551cc256c812fe7651c2a6ebac96 just as suggested by @cdown in #37?

**Comment by juster-0** [comment]
Yes, it occurs after the commit, but only if `enable_alpha=yes`. I also have fixed it in #47.
